packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.v1.vue

```diff
@@ -93,32 +94,144 @@ export default {
         }
+    },
+    computed: {
+        displayModeModifier () {
+            return this.tileMode ? 'c-restaurantCard--tile' : 'c-restaurantCard--list-item';
+        }
     }
 };
 </script>
 
 <style lang="scss" module>
+$card-bgColor                             : $color-container-default;
+$card-borderRadius                        : $radius-rounded-c;
+$tile-imgSpacing                          : spacing(x10) * 2;
+$img-border-radius                        : $radius-rounded-c;
+$img-width                                : 156px;
+$logo-borderRadius                        : $radius-rounded-b;
+$logo-borderColor                         : $color-border-default;
+
+@mixin card-frame {
+    z-index: 1;
+    background: $card-bgColor;
+    box-shadow: 0 6px 8px rgba($color-black, 0.02), 0 1px 20px rgba($color-black, 0.08), 0 3px 6px -1px rgba($color-black, 0.08);
+    border-radius: $card-borderRadius;
+    position: relative;
+}
+
+@mixin restaurantCard-tile {
+    display: block;
+    position: relative;
+    cursor: pointer;
+    padding: spacing(x2);
+    padding-top: spacing() * 3.5;
+
+    .c-restaurantCard-img {
+        height: 228px;
+        width: 100%;
+        left: 0;
+        top: 0;
+        z-index: -1;
+        position: absolute;
+    }
+
+    .c-restaurantCard-content {
+        padding: spacing(x2);
+        @include card-frame;
+    }
+
+    .c-restaurantCard-logo {
+        left: 50%;
+        top: -25px;
+        transform: translateX(-50%);
+    }
+}
 
 .c-restaurantCard {
+    text-decoration: none;
     display: flex;
-    justify-content: center;
-    margin: auto;
-    border: 1px solid purple;
-    font-family: $font-family-base;
-    @include font-size(heading-m);
 }
 
+.c-restaurantCard-logo {
+    border: 1px solid $logo-borderColor;
+    border-radius: $logo-borderRadius;
+    position: absolute;
+
+    .c-restaurantCard--list-item & {
+        top: 50%;
+        transform: translate(-130%, -50%);
+    }
+}
+
+.c-restaurantCard-img {
+    border-radius: $img-border-radius;
+    background-size: cover;
+    background-position: center;
+
+    .c-restaurantCard--list-item & {
+        display: block;
+        flex-basis: $img-width;
+    }
+}
+
+.c-restaurantCard-content {
+    padding: spacing(x0.5) spacing(x5);
+
+    .c-restaurantCard--list-item & {
+        flex: 1;
+        padding-left: spacing(x10);
+    }
+
+    .c-restaurantCard--with-img & {
+        padding-left: spacing(x5);
+    }
+}
+
+.c-restaurantCard--list-item {
+    padding: spacing(x0.5);
+
+    @include media('>mid') {
+        @include card-frame;
+    }
+
+    @include media('<=mid') {
+        @include restaurantCard-tile;
+    }
+}
+
+.c-restaurantCard--tile {
+    @include restaurantCard-tile;
+}
+
+.c-restaurantCard--list-item.c-restaurantCard--with-img {
+    @include media('<=mid') {
+        padding-top: $tile-imgSpacing;
+    }
+}
+
+.c-restaurantCard--tile.c-restaurantCard--with-img {
+    padding-top: $tile-imgSpacing;
+}
```

Comment:
Just to group these a little better under the with/hasImg modifier:

```suggestion
.c-restaurantCard--hasImg {
    &.c-restaurantCard--listItem {
        @include media('<=mid') {
            padding-top: $tile-imgSpacing;
        }
    }

    &.c-restaurantCard--tile {
        padding-top: $tile-imgSpacing;
    }
}
```